taskcluster/kinds/docker-image/kind.yml

```diff
@@ -15,13 +15,15 @@ tasks:
         definition: base-test
         args:
             PYTHON_VERSION: "3.11.9"
+            UV_VERSION: "0.7.15"
 
     # Used by push-image kind
     skopeo: {}
 
     base:
         args:
             PYTHON_VERSION: "3.11.9"
+            UV_VERSION: "0.7.15"
```

Comment:
nit: I think we should just pin to `"{major}.{minor}"` instead of patch. This way we automatically get any bug fixes with it.